### PR TITLE
chore(Tooltip): renamed reference prop to triggerRef

### LINKED ISSUE #304: Tooltip - reference prop renamed to triggerRef
Follow up to breaking change PR https://github.com/patternfly/patternfly-react/pull/8733

Tooltip's reference prop was renamed to triggerRef

_Required actions:_

1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by adamviktora**
Closes #304, relates to https://github.com/patternfly/patternfly-react/pull/8733

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      // No @patternfly/react-core import
+      code: `<Tooltip reference />`,
+    }
+  ],
+  invalid: [
+    {
+      code:   `import { Tooltip } from '@patternfly/react-core'; <Tooltip reference />`,
+      output: `import { Tooltip } from '@patternfly/react-core'; <Tooltip triggerRef />`,
+      errors: [{
+        message: `reference prop has been removed for Tooltip and replaced with the triggerRef prop.`,
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js:19]: This wording needs to match that in the rule

**Review by gitdallas — CHANGES_REQUESTED** [review]

File: packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js (hunk 1 above)
Comment (gitdallas) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js:19]: are you able to run tests locally?

Comment (adamviktora) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/tooltip-rename-reference.js:19]: Yes, thanks! I forgot to run the tests. I wanted to specify a "better" info message but I did not change it in the tests. Should be fixed now (I kept the generated message)

**Review by gitdallas — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]
This looks and works awesomely @adamviktora ! 

I'm also seeing that for issue https://github.com/patternfly/pf-codemods/issues/296, the changes needed will be exactly what you have here, but on the `Popover` component! 

@adamviktora Would you be able to copy these changes over to another branch, rename all `Tooltip` references to `Popover`, and put it up as a PR so we can close out https://github.com/patternfly/pf-codemods/issues/296 as well?

**Comment by jenny-s51** [comment]
@adamviktora great point on the hasCheck-prop-rename rule - I think ideally the codemod should look like hasCheck-prop-rename since we are squashing all Popper and Tooltip `reference` renames from the same react PR https://github.com/patternfly/patternfly-react/pull/8733.

WDYT @gitdallas ?  I see you already approved the popper PR - I think combining them in one PR is probably a good idea to follow our existing conventions, but I'm also not against just merging both.

**Comment by gitdallas** [comment]
@adamviktora @jenny-s51 - yea they could be combined

**Pull request closed without merging**